index.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en">
 <head>
 <meta charset="utf-8" />
-<title>Smashing HTML5!</title>
+<title>Welcome to Bootstrap Starter</title>
```

Comment:
Nice change bro. Could you please Capital case to "to" string please ?

Reply:
i changed this